calculator.py

```diff
@@ -0,0 +1,73 @@
+import sys
+from typing import Counter
+
+
+def count_calls(fn):
+    def _counting(*args, **kwargs):
+        _counting.calls += 1
+        return fn(*args, **kwargs)
+    _counting.calls = 0
+    return _counting
+class Calculator():
+
+    def take_expresion(self):
+        try: 
+            with open(sys.argv[1], 'r') as f:
+                eqesioin = f.read()
+        except IndexError:
+            print('file is not exsist')
+            print("write your expression:")
+            eqesioin = input("> ")
+        return eqesioin
+
+    def calculate(self,eqesioin):
+        arr = []
+        for c in eqesioin:
+            arr.append(c)
+        return self.solver(arr)
+    
+    def solver(self, cal):
+        if len(cal) == 0:
+            return 0
+        stack = []
+        sign = '+'
+        num = 0
+        while len(cal) > 0:
+            c = cal.pop(0)
+            if c.isdigit():
```

Comment:
Try to do this without multiplying by 10 each time. Also, support numbers like `5.65`